doc/libpmem2/pmem2_map.3.md

```diff
@@ -44,24 +44,70 @@ date: pmem2 API version 1.0
 
 # NAME #
 
-**pmem2_map**() - create mapping on the file
+**pmem2_map**() - creates a mapping
 
 # SYNOPSIS #
 
 ```c
 #include <libpmem2.h>
 
-int pmem2_map(const struct pmem2_config *cfg, struct pmem2_map **map_ptr);
+int pmem2_map(const struct pmem2_config *config, struct pmem2_map **map_ptr);
 ```
 
 # DESCRIPTION #
 
-XXX: pmem2_map description
+The **pmem2_map**() function maps a file to the process virtual address space.
+It uses the values, stored in the *config* object, as parameters describing
+how to perform the mapping:
+
```

Comment:
You should group up config values by type and which configs exclude one another.
For example, these two define the source of the mapping, and are mutually exclusive. Another one will be DRAM.